docs/versioned_docs/version-1.0.x/examples/custom-tabbed-header.md

```diff
@@ -0,0 +1,220 @@
+    paddingHorizontal: 12,
+    paddingVertical: 8,
+  },
+  tabWrapperStyle: {
+    paddingVertical: 10,
+  },
+  tabsContainerStyle: {
+    paddingHorizontal: 10,
+  },
+})
+
+return (
+  <TabbedHeaderPager
+    //...
+    tabsContainerBackgroundColor={colors.black}
+    tabTextContainerStyle={styles.tabTextContainerStyle}
+    tabTextContainerActiveStyle={styles.tabTextContainerActiveStyle}
+    tabTextStyle={styles.tabText}
+    tabTextActiveStyle={styles.tabTextActiveStyle}
+    tabWrapperStyle={styles.tabWrapperStyle}
+    tabsContainerStyle={styles.tabsContainerStyle}
+    //...
+  >
+    {/** content */}
+  </TabbedHeaderPager>
+)
+```
+
+## Custom header foreground
+
+Instead of setting header color by `backgroundColor` prop, we can use `backgroundImage` and pass an image.
+In the TabbarHeader example there is a small avatar image and title below, we can set the first one by passing
+image to the `foregroundImage` prop.
+To customise title, we can pass it by title prop `title="Baby Yoda"` and then pass styles by `titleStyle` prop.
+
+```tsx
+const styles = StyleSheet.create({
+  titleStyle: {
+    color: 'white',
+    fontWeight: 'bold',
+    padding: 10,
+    fontSize: 40,
+    backgroundColor: 'rgba(0,0,0,0.6)'
+  }
+})
+
+return (
+  <TabbedHeaderPager
+    // ...
+    backgroundImage={{
+      uri: 'https://yoda.jpeg',
+    }}
+    title="Baby Yoda"
+    titleStyle={styles.titleStyle}
+    foregroundImage={{
+      uri: 'https://starwars.png',
+    }}
+    // ...
+  >
+    {/** content */}
+  </TabbedHeaderPager>
+)
+```
+## Custom Header component
+
+Instead of passing your own logo to the header, you can create component and pass it to the `renderHeaderBar` prop. It allows you to create back/close button and create custom animations
+
+In the example below there is custom header containing close button which is visible all the time, 
+and the title displayed on header, visible only when the header is in closed state.
+
+In order to do this, we save vertical content offset in a Reanimated's shared value and then use it to interpolate opacity value of the View with the title.
+
+```tsx
+const HeaderBar = ({ scrollValue }) => {
+  //...
+
+  const animatedStyle = useAnimatedStyle(() => {
+    return { opacity: interpolate(scrollValue.value, [0, 60, 90], [0, 0, 1], Extrapolate.CLAMP) };
+  });
+
+  return (
+    <SafeAreaView edges={['top', 'left', 'right']} style={...}>
+      <View style={...}>
+        <TouchableOpacity onPress={...}>
+          <Image
+            style={...}
+            resizeMode="contain"
+            source={{
+              uri: 'https://close.png',
+            }}
+          />
+        </TouchableOpacity>
+        <Animated.View style={animatedStyle}>
+          <Text style={...}>Baby Yoda</Text>
+        </Animated.View>
+      </View>
+    </SafeAreaView>
+  );
+};
+
+const YodaScreen = () => {
+  // ...
+  const scrollValue = useSharedValue(0);
+
+  function onScroll(e: NativeScrollEvent) {
```

Comment:
It's a good practice to name handlers with `handle` prefix. `on` should be used to name props, not handlers.